presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -49,14 +53,18 @@ public void setUp()
             throws Exception
     {
         database = new TestingDatabase();
-        cachingJdbcClient = createCachingJdbcClient(true);
+        cachingJdbcClient = createCachingJdbcClient(true, true);
```

Comment:
`createCachingJdbcClient(CACHE_MISSING, CACHE_ENABLED)`? Does it make sense to have cache with cache missing enabled and entire cache disabled? Maybe you could have 3 dedicated methods instead of `true, true` paremeters? Or instead of `cachingEnabled` you can pass `FOREVER` or `NEVER`. Also we are missing tests for refreshes (`REFRESH_AFTER_100_MS`).